eclass/toolchain-funcs.eclass

```diff
@@ -214,7 +214,7 @@ tc-cpp-is-true() {
 		EOF
 	)
 
-	[[ ${RESULT} == true ]]
+	[[ $(echo "${RESULT}" | tr -d '[[:space:]]') == true ]]
```

Comment:
Calling `tr` seems a bit big for this. Why not just match `*true*`?

Also, changes to widely used eclasses should go as git patches to gentoo-dev mailing list (you need to be subscribed before posting).

Reply:
I originally had *true*, but then vapier@ made me switch to this ;-) See https://chromium-review.googlesource.com/c/chromiumos/overlays/portage-stable/+/1855039/2#message-cb0449c47e121902d1b764f14fbdd270c4f92ec2 for more context.

Given the discussion on the bug, it looks like we should ignore even more stray output. I like the idea to generate an #error and rely on the preprocessor exit status instead, so I'll switch the code to do that.

Sent to gentoo-dev@ as well just now.